### ISSUE TITLE
Color contrast issues with `views-exposed-form` 

### ISSUE THREAD

**Issue by bspeare** [uiowa/uiowa#5109]
# Problem
As @joewhitsitt noted in https://github.com/uiowa/uiowa/pull/4479#issuecomment-1091938819, the views exposed form has color contrast issues with the form labels.  

# Steps to reproduce
Add `bg--black` to a surrounding `div` on https://sandbox.prod.drupal.uiowa.edu/sandbox-people.  

# Proposed solution
- Remove fixed background color in https://github.com/uiowa/uiowa/blob/main/docroot/themes/custom/uids_base/scss/views/views.scss#L9. 
- Add `bg--gray` class via preprocess function to the views-exposed-form.
